src/Symfony/Bridge/Doctrine/composer.json

```diff
@@ -17,9 +17,13 @@
     ],
     "require": {
         "php": "^7.1.3",
-        "doctrine/common": "~2.4",
         "symfony/polyfill-ctype": "~1.8",
-        "symfony/polyfill-mbstring": "~1.0"
```

Comment:
can you move the doctrine/* packages before the symfony one, to have alpha order please?

Reply:
If you agree, I will send separate PR with this and enabling `sort-packages` config option - the order is inconsistent also for many non-doctrine packages so I'd prefer not to mix it here. :relaxed:

Reply:
sort-package doesn't work here, because we manually merge composer.json files, so no need, as that would only make mergers job a bit harder/slower